blivet/udev.py

```diff
@@ -592,15 +597,12 @@ def device_get_lv_type(info):
 
 def device_dm_subsystem_match(info, subsystem):
     """ Return True if the device matches a given device-mapper subsystem. """
-    uuid = info.get("DM_UUID", "")
-    uuid_fields = uuid.split("-")
-    _subsystem = uuid_fields[0]
-    if _subsystem.lower().startswith("part") and len(uuid_fields) > 1:
-        # kpartx uses partN- as a subsystem prefix, which we ignore because
-        # we only care about the subsystem of the partitions' parent device.
-        _subsystem = uuid_fields[1]
-
-    if _subsystem == uuid or not _subsystem:
+    name = info.get("DM_NAME", None)
```

Comment:
`None` is the default return value for `dict.get` when called with a missing key, so you can just use `info.get("DM_NAME")` there.

Reply:
Fixed.